docs/tutorials/preview-monokle-environment-settings.md

```diff
@@ -0,0 +1,61 @@
+# Preview Monokle Environment Settings
+
+Monokle is a fully integrated IDE for managing manifests. It provides instant access for debugging Kubernetes resources without looking into the yaml syntax. 
+
+In this tutorial, we have illustrated the steps to set up Monokle Environment. 
```

Comment:
".. to configure Monokle"

We should also note that these values are initialised by default - there is no immediate need to change them.

Reply:
Can please be specific with "these values". Do you mean K8 resources?

Reply:
no - I meant the various settings in the settings panel

Reply:
Understood

Reply:
Done